templates/datalab/auth0-configmap.template.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: auth0-config
+  namespace: {{ namespace }}
+data:
+  config: |
+    {
+      "audience": "https://datalab-api.datalabs.nerc.ac.uk/",
```

Comment:
Should this be something to do with `testlab`? Or does the audience Auth0 stuff so is looking at something else?